packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table/use-row-data.ts

```diff
@@ -18,21 +17,21 @@ export const useRowData = () => {
     return generatePropertyRowsFromEntity(rootEntityAndSubgraph);
   }, [rootEntityAndSubgraph]);
 
-  const sortedRowData = useMemo(
-    () => sortRowData(rowData, propertySort),
-    [rowData, propertySort],
-  );
+  const flattenRowData = useCallback(
+    (sortedRowData: RowData) => {
+      const propertyRows = sortedRowData as PropertyRow[];
```

Comment:
I think it's probably better to just type this properly, and if casting is necessary, to do it where it's being called. But i think as mentioned in another comment it may be possible to use generics so you don't need to cast.

```suggestion
    (propertyRows: PropertyRow[]) => {
```